Probesys/bileto
# New/update password

Related to https://github.com/Probesys/bileto/issues/90

Changes proposed in this pull request:

- Add current / new passwords fields to the profile page
- Create a Stimulus controller to allow to show / hide password

How to test the feature manually:

1. go to the profile page
2. change your password
3. verify that the show/hide buttons work
4. verify you can login with your new password

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._
